sh/arquillian-search-tests-run-all.sh

```diff
@@ -134,7 +134,10 @@ test_run dynamic-data-lists/dynamic-data-lists-test \
 	*.search.*Test
 
 test_run layout/layout-test \
-	*.search.*Test
+	*.search.*Test \
```

Comment:
Remove line with `search` package; leave only the lines with class names. The package contains the defective test classes we want to quarantine, so we need to remove it and run only the green classes (until we fix them all)

This is correct: https://github.com/liferay-search/liferay-search-tck/pull/1#issuecomment-832252776

Reply:
Ok, thanks @arboliveira ! I will do that.